### ISSUE TITLE
PyPDF2.errors.DeprecationError: PdfFileReader is deprecated and was removed in PyPDF2 3.0.0. Use PdfReader instead.

### ISSUE THREAD

**Issue by liar666** [caizefeng/html2pdf_bookdown#1]
Does not work anymomre due to following error :'(

```
Traceback (most recent call last):
  File "/home/guillaume.muller/Python/Html2PdfBookdown/bin/html2pdf_bookdown", line 8, in <module>
    sys.exit(main())
  File "/home/guillaume.muller/Python/Html2PdfBookdown/lib/python3.10/site-packages/html2pdf_bookdown/__main__.py", line 19, in main
    contents_list_modified = print_to_pdf(contents_list, args.book_name)
  File "/home/guillaume.muller/Python/Html2PdfBookdown/lib/python3.10/site-packages/html2pdf_bookdown/convert.py", line 40, in print_to_pdf
    page_num = PdfFileReader(f).getNumPages()
  File "/home/guillaume.muller/Python/Html2PdfBookdown/lib/python3.10/site-packages/PyPDF2/_reader.py", line 1974, in __init__
    deprecation_with_replacement("PdfFileReader", "PdfReader", "3.0.0")
  File "/home/guillaume.muller/Python/Html2PdfBookdown/lib/python3.10/site-packages/PyPDF2/_utils.py", line 369, in deprecation_with_replacement
    deprecation(DEPR_MSG_HAPPENED.format(old_name, removed_in, new_name))
  File "/home/guillaume.muller/Python/Html2PdfBookdown/lib/python3.10/site-packages/PyPDF2/_utils.py", line 351, in deprecation
    raise DeprecationError(msg)
PyPDF2.errors.DeprecationError: PdfFileReader is deprecated and was removed in PyPDF2 3.0.0. Use PdfReader instead.
```

**Comment by liar666** [comment]
Following the error, I tried to change `PdfFileReader` to `PdfReader` in `html2pdf_bookdown/convert.py`::40 , but then I get:
```Traceback (most recent call last):
  File "/home/guillaume.muller/Python/Html2PdfBookdown/bin/html2pdf_bookdown", line 8, in <module>
    sys.exit(main())
  File "/home/guillaume.muller/Python/Html2PdfBookdown/lib/python3.10/site-packages/html2pdf_bookdown/__main__.py", line 19, in main
    contents_list_modified = print_to_pdf(contents_list, args.book_name)
  File "/home/guillaume.muller/Python/Html2PdfBookdown/lib/python3.10/site-packages/html2pdf_bookdown/convert.py", line 40, in print_to_pdf
    page_num = PdfReader(f).getNumPages()
NameError: name 'PdfReader' is not defined. Did you mean: 'PdfFileReader'?
```
WTF!?!
Sorry, I'm not enough Python-versed to go further...

**Comment by liar666** [comment]
Woaw yeah sorry, you need to change the  `import` also.

Then you get:
```
PyPDF2.errors.DeprecationError: reader.getNumPages is deprecated and was removed in PyPDF2 3.0.0. Use len(reader.pages) instead.
```
If you change that also it works :)

I'll try to convert that into a PR...

**Comment by liar666** [comment]
There are a few more changes to apply.

I finally managed to do the changes myself thanks to the guidance of PyPDF2 3.0.0 itself.

In the end:

`PyPDF2.errors.DeprecationError: PdfFileReader is deprecated and was removed in PyPDF2 3.0.0. Use PdfReader instead.` (import + object creation)

`PyPDF2.errors.DeprecationError: reader.getNumPages is deprecated and was removed in PyPDF2 3.0.0. Use len(reader.pages) instead.`

`PyPDF2.errors.DeprecationError: PdfFileMerger is deprecated and was removed in PyPDF2 3.0.0. Use PdfMerger instead.` (import + object creation)

`PyPDF2.errors.DeprecationError: addBookmark is deprecated and was removed in PyPDF2 3.0.0. Use add_outline_item instead.`

See my PR
